recipe/meta.yaml

```
@@ -16,20 +16,26 @@ requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('rust') }}
-    - {{ stdlib("c") }}
     - cargo-bundle-licenses
 
 test:
+  requires:
+    - pip
   commands:
     - just --help
+    - pip check
```

```suggestion
  commands:
    - just --help
```
`just` doesn't need or use python.